sbc_compassion/tools/bluecornerfinder.py

```diff
@@ -119,42 +116,8 @@ def _readBlueCorner(self):
                 # check if a good blue pixel is found
                 if checkColor(self.img[tuple(ind)], self.threshold):
                     if self._checkNext(ind):
-                        self.found = True
-
-        # case where a blue pixel has been found
-        if self.found:
-            self._findmin()
-        else:
-            self.ind = None
-
-        return self.ind
-
-    def _findmin(self):
```

Comment:
I think we should keep this function in order to take the closest pixel and not one next to it